install/yamls/ccruntime-peer-pods-in-pod.yaml

```diff
@@ -31,6 +31,9 @@ spec:
             name: peer-pods-cm
         command: ["/usr/local/bin/entrypoint.sh"]
         volumeMounts:
+        - name: auth-json
+          mountPath: "/root/containers/" # hardcoded
```

Comment:
Should it be full path like `/root/containers/auth.json` ?

Reply:
i thought so as well but then it was mounted at `/root/containers/auth.json/auth.json `:P